src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -639,7 +676,14 @@ private void replaceEC2Servers() {
                     String id = job.getInstanceId();
                     LOG.warn("Error encountered while monitoring server {}. Terminating.", id);
                     remainingInstances.removeIf(instance -> instance.getInstanceId().equals(id));
-                    ServerController.terminateInstances(ec2, id);
+                    try {
+                        ServerController.terminateInstances(getEC2ClientForDeployJob(), id);
```

Comment:
Why is this using the ServerController static method to terminate instances? Needs a note to explain.